src/wp-includes/user.php

```diff
@@ -1587,6 +1592,12 @@ function clean_user_cache( $user ) {
 	 * @param WP_User $user    User object.
 	 */
 	do_action( 'clean_user_cache', $user->ID, $user );
+
+	if ( get_current_user_id() === (int) $user->ID ) {
+		$user_id = (int) $user->ID;
+		$current_user = null;
+		wp_set_current_user( $user_id, '' );
```

Comment:
- Align the variable assignments
- Consider adding an inline comment for educational purpose and quicker understanding. Though in the DocBlock, it may not be quickly obvious what this is doing and why.

```suggestion
	// Refresh the global user instance if the cleaning current user.
	if ( get_current_user_id() === (int) $user->ID ) {
		$user_id      = (int) $user->ID;
		$current_user = null;
		wp_set_current_user( $user_id, '' );
```